### PR TITLE
Add same information from GH readme to RTD docs

### PR DESCRIPTION
**Pull request by kinow**
Hello,

I was working on some review feedback in an eFlows4HPC, where we were asked to add the acronym of FESOM in the document.

I accessed the RTD site but I couldn't find it. I found it later on fesom.de. The GitHub README has this information, so I copied that over here, along with the introductory paragraph from GH's README.

Local preview:

image

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -6,6 +6,15 @@
 FESOM2 documentation
 ====================
 
+The Finite Element Sea Ice-Ocean Model (FESOM2).
```
Comment (koldunovn) [inline docs/index.rst:9]: The FESOM 2 is finite-volume version, so: "Finite volumE Sea Ice-Ocean Model (FESOM2)"

File: docs/index.rst [hunk 2]
```diff
@@ -6,6 +6,15 @@
...
+
+Multi-resolution ocean general circulation model that solves
+the equations of motion describing the ocean and sea ice using
+finite-element and finite-volume methods on unstructured
```
Comment (koldunovn) [inline docs/index.rst:13]: remove finite-element, that's for FESOM 1 :)

**Comment by koldunovn** [comment]
Thanks a lot, @kinow , it totally make sense! I made a couple of suggestions, so things are valid for FESOM2 :)

File: docs/index.rst (hunk 1 above)
Comment (kinow) [reply docs/index.rst:9]: Done!

File: docs/index.rst (hunk 2 above)
Comment (kinow) [reply docs/index.rst:13]: Done (updated paragraphs to balance the text & linebreaks)

**Comment by kinow** [comment]
> Thanks a lot, @kinow , it totally make sense! I made a couple of suggestions, so things are valid for FESOM2 :)

Thanks @koldunovn for the review and explanation on the different meaning for FESOM1 and FESOM2. I think it could be useful to have that too somewhere in the docs (if not already, like in a "History" section). But probably for another issue. Edited commit and push forced with the changes. Thanks!!!

**Review by koldunovn — APPROVED** [review]

**Pull request merged**